projects/robots/bitcraze/crazyflie/controllers/crazyflie_py_wallfollowing/crazyflie_py_wallfollowing.py

```diff
@@ -0,0 +1,191 @@
+#  ...........       ____  _ __
+#  |  ,-^-,  |      / __ )(_) /_______________ _____  ___
+#  | (  O  ) |     / __  / / __/ ___/ ___/ __ `/_  / / _ \
+#  | / ,..´  |    / /_/ / / /_/ /__/ /  / /_/ / / /_/  __/
+#     +.......   /_____/_/\__/\___/_/   \__,_/ /___/\___/
+
+# MIT License
+
+# Copyright (c) 2022 Bitcraze
+
+# @file crazyflie_controllers_py.py
+# Controls the crazyflie motors in webots in Python
+
+"""crazyflie_py_wallfollowing.py controller."""
+
+
+from controller import Robot
+from controller import Keyboard
+
+from math import cos, sin
+
+from pid_controller import pid_velocity_fixed_height_controller
+from wall_following import WallFollowing
+
+FLYING_ATTITUDE = 1
+
+if __name__ == '__main__':
+
+    robot = Robot()
+    timestep = int(robot.getBasicTimeStep())
+
+    # Initialize motors
+    m1_motor = robot.getDevice("m1_motor")
+    m1_motor.setPosition(float('inf'))
+    m1_motor.setVelocity(-1)
+    m2_motor = robot.getDevice("m2_motor")
+    m2_motor.setPosition(float('inf'))
+    m2_motor.setVelocity(1)
+    m3_motor = robot.getDevice("m3_motor")
+    m3_motor.setPosition(float('inf'))
+    m3_motor.setVelocity(-1)
+    m4_motor = robot.getDevice("m4_motor")
+    m4_motor.setPosition(float('inf'))
+    m4_motor.setVelocity(1)
+
+    # Initialize Sensors
+    imu = robot.getDevice("inertial unit")
+    imu.enable(timestep)
+    gps = robot.getDevice("gps")
+    gps.enable(timestep)
+    gyro = robot.getDevice("gyro")
+    gyro.enable(timestep)
+    camera = robot.getDevice("camera")
+    camera.enable(timestep)
+    range_front = robot.getDevice("range_front")
+    range_front.enable(timestep)
+    range_left = robot.getDevice("range_left")
+    range_left.enable(timestep)
+    range_back = robot.getDevice("range_back")
+    range_back.enable(timestep)
+    range_right = robot.getDevice("range_right")
+    range_right.enable(timestep)
+
+    # Get keyboard
+    keyboard = Keyboard()
+    keyboard.enable(timestep)
+
+    # Initialize variables
+
+    past_x_global = 0
+    past_y_global = 0
+    past_time = 0
+    first_time = True
+
+    # Crazyflie velocity PID controller
+    PID_CF = pid_velocity_fixed_height_controller()
```

Comment:
We can accept the PID abbreviation because it is a very common one. However, the CF abbreviation is not clear and should be either removed or expanded.
```suggestion
    PID_CF = pid_velocity_fixed_height_controller()
```